.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,8 @@ Fixes / closes #????
 - [ ] Added / updated test-suite
 <!-- If this is a feature pull request / breaks compatibility: -->
 <!-- (Otherwise, remove these lines.) -->
-- [ ] Corresponding documentation was added / updated (including any warning and error messages added / removed / modified).
+- [ ] If you modified Coq's grammar, run `make doc_gram_rsts` to update the syntax in the documentation (see `docgram/README.md`).
+- [ ] Corresponding documentation was added / updated, including any changed warning and error messages.
 - [ ] Entry added in the changelog (see https://github.com/coq/coq/tree/master/doc/changelog#unreleased-changelog for details).
```

Comment:
We could try to be more consistent with the rest of the template (where most conditions are inside comments).
Also note that inserting this new checkbox just after "if this is a feature PR" doesn't really make sense. Let's have it with its own condition block.

```suggestion
- [ ] Corresponding documentation was added / updated, including any changed warning and error messages.
- [ ] Entry added in the changelog (see https://github.com/coq/coq/tree/master/doc/changelog#unreleased-changelog for details).
<!-- If you modified Coq's grammar (otherwise remove this line): -->
- [ ] Run `make doc_gram_rsts` to update the syntax in the documentation (see `docgram/README.md`).
```

Reply:
OK, but I would put the doc_gram item before updating the doc--makes more sense to do the items in that order.  And maybe have updating the changelog first, since most but not all PRs change it.

OTOH, I wonder how many people read this for every PR they submit?  I've gotten to the point where deleting the template is a routine first step for my PRs.

Reply:
> OTOH, I wonder how many people read this for every PR they submit? I've gotten to the point where deleting the template is a routine first step for my PRs.

Well, me too also I will sometimes keep the checkbox items that interest me. And the thing being there seems to be sufficient to actually make people do the requested steps (or at least it was, see Section 3.5.2 from [my thesis](https://www.irif.fr/_media/users/theo/memoirthesis.pdf)).